src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -197,6 +177,67 @@ public async Task<bool> LaunchTestHostAsync(TestProcessStartInfo testHostStartIn
         };
     }
 
+    private string GetTestHostName(Architecture architecture, Framework targetFramework, PlatformArchitecture processArchitecture)
+    {
+        string testHostProcessName = "testhost";
```

Comment:
[minor] It might be better to use a `StringBuilder` as we do multiple concat.

Reply:
this happens once per testhost, and is just a few tiny strings, but I did it anyway, the code looks nicer. 👍